From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: ViKing <viking.yfy@qq.com>
Date: Sat, 29 Aug 2026 20:31:00 +0800
Subject: [PATCH] net: dsa: qca8k: add RE-CS-08 firmware debug tag

Print a single temporary build tag when the QCA8386 match data is selected.
This patch is the only RE-CS-08 firmware marker and is refreshed in place
for later test images.

Signed-off-by: ViKing <viking.yfy@qq.com>
---
 drivers/net/dsa/qca/qca8k-8xxx.c | 3 +++
 1 file changed, 3 insertions(+)

--- a/drivers/net/dsa/qca/qca8k-8xxx.c
+++ b/drivers/net/dsa/qca/qca8k-8xxx.c
@@ -2323,6 +2323,9 @@ qca8k_sw_probe(struct mdio_device *mdiod
 	priv->bus = mdiodev->bus;
 	priv->dev = &mdiodev->dev;
 	priv->info = of_device_get_match_data(priv->dev);
+	if (qca8k_is_qca8386(priv))
+		dev_info_once(priv->dev,
+			      "RE-CS-08 firmware debug tag: 20260829-ppe-fusion-r1\n");
 
 	priv->reset_gpio = devm_gpiod_get_optional(priv->dev, "reset",
 						   GPIOD_OUT_HIGH);
